app/tags/[tagId]/page.tsx

```diff
@@ -41,3 +41,9 @@ export default async function Page(props: Props) {
     </>
   );
 }
+
+export async function generateStaticParams() {
+  const contentIds = await client.getAllContentIds({endpoint: 'tags'})
```

Comment:
```suggestion
  const tagIds = await getTagList();
```

microcms.ts にタグ取得用の関数があるためそちらを用いたほうが見やすいと感じました